Dockerfile.arm64

```diff
@@ -0,0 +1,17 @@
+FROM lopsided/archlinux
+LABEL maintainer="Robin Cole @robmarkcole"
```

Comment:
If you are happy to own this, please add your email and I can add you as a contributor

Reply:
Totally up for it. You want my proper email in there or just Name and github handle?

Reply:
Thats your call. Beware once your email is out there all sort of people may contact you!

Reply:
Lol, yeah, we'll leave it at my docker/gihub handle